packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -704,6 +705,170 @@ describe('TransactionController', () => {
     });
   });
 
+  describe('with actionId', () => {
+    it('adds multiple transactions with same actionId and ensures second transaction result does not resolves before the first transaction result', async () => {
```

Comment:
Can we split this into two tests as in the extension?

The first ignores the `result` promise and just checks the number of transactions in the state, where as the second would only assert the results are resolved together?

Reply:
Perfect! I've split the tests.